### PR TITLE
Fix HD selection in UI

### PR DESCRIPTION
**Pull request by haslinghuis**
MSP call was corrupting firmware config (removing HD option).

Wrapped the call to `MSP_OSD_CANVAS` to be handled before calling updateOsdView seems to fix it.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
